shared/src/business/useCases/pdf/sanitizePdfInteractor.js

```diff
@@ -1,10 +1,27 @@
+/**
+ * setPdfFields helper function
+ *
+ * @param {array} fields array of pdf form fields
+ */
+
+exports.setPdfFields = fields => {
```

Comment:
I love that this was added as a use case helper! 

Nit: Could it be extracted into its own file in the `useCaseHelpers` folder?

Reply:
Let's do it